src/test/api.test.js

```diff
@@ -125,23 +125,28 @@ tap.test('controller/api', async (t) => {
 
     t.test('when the request is good', async (t) => {
       const body = Object.assign({}, config[0]);
+      const CLIENTID = '75edb050796a0eb1cf2cfb0da7245f85bc50baa7';
```

Comment:
clientID would be preferred to caps, same with provider.

Reply:
Thought of it being a constant which could be inferred from the capitalization.

Reply:
Would make sense for global constants, you could move them to the top of the file and move other usages to caps if you like.